libs/Matchmaking/test/Matchmaking.PeerDiscovery.Test/Matchmaking.PeerDiscovery.Test.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+    <UseNugetPackages Condition="'$(UseNugetPackages)'==''">$(Configuration.EndsWith('nuget'))</UseNugetPackages>
```

Comment:
Projects which use local or nuget projects would have to repeat this kind of boilerplate.
This is probably (?) only for test and demo projects.

Reply:
Only tests I'd say, I am not sure demos need to be switched back and forth.